packages/dx-react-grid-bootstrap3/src/templates/table-header-cell.jsx

```diff
@@ -51,6 +51,10 @@ export class TableHeaderCell extends React.PureComponent {
           ...(dragging || (tableColumn && tableColumn.draft) ? { opacity: 0.3 } : null),
           ...style,
         }}
+        ref={node => node && getCellWidth(() => {
+          const { width } = node.getBoundingClientRect();
```

Comment:
According React concepts we should use DOM inside the `componentDidMount` method.

Reply:
You can use `ref` and in the `componentDidMount` (or similar for function components) use DOM operations.

About legacy code... maybe we should override it?)

Reply:
Understood, I'll do it